test/llvmpasses/Makefile

```diff
@@ -1,8 +1,14 @@
 SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME := $(abspath $(SRCDIR)/../..)
 include $(JULIAHOME)/Make.inc
-test:
+
+check: $(SRCDIR)
```

Comment:
update the CI configuration that runs these tests to use the `check` target instead?

Reply:
We don't seem to be hardcoding the name anywhere. Do you know of a case?

Reply:
Fair, I guess this is the default target, so things will keep working.